docs/emr_deployment.md

```diff
@@ -0,0 +1,126 @@
+---
+id: emr_deployment
+title: GeoTrellis EMR deployment with GDAL
+sidebar_label: GeoTrellis EMR deployment with GDAL
+---
+
+GeoTrellis can be used with _EMR 6_. However, if there is a need in GDAL bindings usage there can be some certain complexities.
+This section describes the entire _EMR 6_ deployment and GDAL installation in a way that would satisfy GeoTrellis GDAL requirements.
+
+Starting _GeoTrellis_ `3.4.2`, _GeoTrellis_ is compatible only with the GDAL`3.1 +` versions.
+
+## Installing GDAL on EMR
+
+The EMR installation conatans of the following steps:
```

Comment:
IIUC, the goal of this page is to get a user setup with a working GDAL installation on EMR as quickly as possible. Instead, this section is describing what the bootstrap script is doing.  In order to meet then aforementioned goal, what would you think about swapping this section to briefly note the three options we provide (bootstrap script, g8 template, sbt lighter example) and then describe each in more detail in their respective subheadings? That way, right at the top of the document, the user can see at a glance what their options are and then scroll for more.

I think this would mostly just be streamlining and reorganizing the content that's already here.